### DISCUSSION TITLE
How to introduce custom container as honeypot in T-Pot installation

### DISCUSSION THREAD

**Discussion by kumbhapr** [telekom-security/tpotce#1691] [Q&A]
We want to introduce a custom component (microservice APIs) as honeypot to expose via T-Pot platform and collect logs, activities for that honeypot. 
I searched documentation of T-Pot but couldn't find reference on how such custom service as honeypot can be added to T-Pot platform.
Can someone provide insights if know on how to achieve such customization ?

Regards
Prasad

**Comment by dmille6** [comment]
you'd just build your docker container, then add it to the main docker compose for the tpot
you'd want to make sure it communicated with the tpot containers through the set ports and networks. 

i have a python script I just through on the honeypot that runs via cron, that pulls the data I want
